src/card/view/CardViewHeader.tsx

```diff
@@ -147,6 +158,7 @@ const NeoCardViewHeader = ({
             {downloadImageEnabled ? downloadImageButton : <></>}
             {fullscreenEnabled ? expanded ? unMaximizeButton : maximizeButton : <></>}
             {descriptionEnabled ? descriptionButton : <></>}
+            {refreshButton}
```

Comment:
i think we should make the refresh button optional (default: disabled) like all other card header buttons. It gives for a quite busy view otherwise.

Probably also we should hide it for non-reports like the iFrame, Markdown and Parameter Selector.